src/typescript/sdk/src/indexer-v2/types/index.ts

```diff
@@ -595,6 +595,18 @@ export const toUserPoolsRPCResponse = (data: DatabaseJsonType["user_pools"]) =>
   dailyVolume: BigInt(data.daily_volume),
 });
 
+const unq64 = (n: number) => n / 2 ** 64;
+
+export const toPriceFeedRPCResponse = (data: DatabaseJsonType["price_feed"]) => ({
+  marketID: data.market_id,
+  symbolBytes: data.symbol_bytes,
+  symbolEmojis: data.symbol_emojis,
+  marketAddress: data.market_address,
+  openPrice: unq64(data.open_price_q64),
+  closePrice: unq64(data.close_price_q64),
+  deltaPercentage: (1 - unq64(data.close_price_q64) / unq64(data.open_price_q64)) * 100,
```

Comment:
```suggestion
  deltaPercentage: Big(1)
    .minus(q64ToBig(data.close_price_q64).div(q64ToBig(data.open_price_q64)))
    .mul(100);
```

Since these are numeric types let's be safe and use `Big` for this